arcaflow_plugin_fio/fio_plugin.py

```diff
@@ -17,6 +17,23 @@
 )
 
 
+def split_json_and_errors(fio_output: str):
+    # Sometimes fio informational messages are included at the top of the JSON output.
+    # Separate the json data from the errors and return them as a tuple.
+    error_data = ""
+    lines = fio_output.splitlines()
+    for i in range(len(fio_output)):
+        file_data = "\n".join(lines[i:])
+        try:
+            json_data = json.loads(file_data)
+            break
+        except json.JSONDecodeError:
+            error_data += f"{lines[i]}\n"
+            continue
+
+    return json_data, error_data
```

Comment:
I don't love that this needs to be done like this. But I don't think another requirement, like checking for it starting with `{` would gain us much.

Reply:
It's a little ugly, but it's the same approach used by fio in their own tests.